# This message describes a log provided by the watchdog node
#

# log types
byte INFO=0
byte ERROR=1
byte WARN=2

byte type
string msg
